datahub-graphql-core/src/main/resources/gms.graphql

```diff
@@ -3,7 +3,7 @@ scalar Long
 
 schema {
     query: Query
-    mutation: Mutation
+#    mutation: Mutation
```

Comment:
Is there a reason we need to comment this out? It will break the `datahub-frontend` server (which extends the Mutation type) if we do

Reply:
Currently the mutation type has no fields I get a compilation error.

```
nested exception is graphql.schema.validation.InvalidSchemaException: invalid schema:
"Mutation" must define one or more fields.
```

Reply:
Ah got it, okay